client/layout/sidebar/item.jsx

```diff
@@ -68,7 +68,7 @@ export default React.createClass( {
 					<span className="menu-link-text">{ this.props.label }</span>
 					{ isExternalLink ? <span className="noticon noticon-external" /> : null }
 				</a>
-				{ this.renderButton( this.props.buttonLink ) }
+				{ this.renderButton( this.props.buttonLink ) || this.props.children }
```

Comment:
Haha, yes. I left it while I was working through the sidebar, should be safe to remove now.